packages/shared/ReactTypes.js

```diff
@@ -234,3 +234,21 @@ export type MutableSource<Source: $NonMaybeType<mixed>> = {|
   _currentPrimaryRenderer?: Object | null,
   _currentSecondaryRenderer?: Object | null,
 |};
+
+// The subset of a Thenable required by things thrown by Suspense.
+// This doesn't require a value to be passed to either handler.
+export interface Wakeable {
```

Comment:
Thenable is called that way because you can call then on it.

Maybe this should be Waker or something.

Reply:
I guess in Rust, a Waker is the thing you call to then (essentially). It's the thing to be woken up.

So I guess Wakeable is fine. It's the thing that will at some point trigger to wake up the Waker.